File: example/src/main/java/com/genius/groupie/example/item/SwipeToDeleteItem.java
package com.genius.groupie.example.item;

import android.support.annotation.ColorRes;
import android.support.v7.widget.helper.ItemTouchHelper;

public class SwipeToDeleteItem extends CardItem {
    public SwipeToDeleteItem(@ColorRes int colorRes) {
        super(colorRes);
    }

    @Override public int getSwipeDirs() {
        return ItemTouchHelper.LEFT | ItemTouchHelper.RIGHT;
    }
}

File: example/src/main/java/com/xwray/groupie/example/item/SwipeToDeleteItem.kt
package com.xwray.groupie.example.item

import androidx.annotation.ColorRes
import androidx.recyclerview.widget.ItemTouchHelper

class SwipeToDeleteItem(@ColorRes colorRes: Int) : CardItem(colorRes) {

    override val swipeDirs: Int = ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT
}
